src/models/subgraph-entities/vX/project.ts

```diff
@@ -24,6 +24,9 @@ export type Project = {
   projectId: number
   pv: PV
   owner: string
+  creator: string
+  contributorsCount: number
+  nftsMintedCount: number
```

Comment:
subgraph entities will be deprecated soon, but to avoid making this PR any thicker I've just updated the current subgraph Project entity type for compatibility, which is still used in the DB update routine